frontend/src/Components/Footer.jsx

```diff
@@ -57,7 +58,6 @@ const Footer = () => {
     };
 
     try {
-      //This is the URL you can use if you are working on local machine if creating error in finding backendURL so use this in fetch request = 'http://localhost:8080/api/feedback
       const response = await fetch(`${backendURL}/api/feedback`, {
```

Comment:
@devxMani why remove this comment?
You have removed all the comments wherever it wsa before. Please put those comments back

Reply:
@AlfiyaSiddique i'll create a new pr for this